frontend/src/pages/CourseEditor/LessonsStep/LessonItem/LessonItem.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC } from 'react';
+
+import {
+  LessonItemWrapper,
+  LessonItemTitle,
+  LessonInput,
+  InputSelect,
+  SelectItem,
+  InputBox,
+  InputLabel,
+  InputText,
+  InputTextArea,
+} from './styled';
+
+import { ILessonItemProps } from '../../types';
+
+const LessonItem: FC<ILessonItemProps> = ({ handleChangeOption, selectOption, formik }) => (
+  <LessonItemWrapper>
+    <LessonItemTitle>Lesson № 1</LessonItemTitle>
+    <form onSubmit={formik.onSubmit}>
```

Comment:
you don't need to use form here and to add onSubmit here, onSubmit will be able only on the last stage of our form